eclass/debian.eclass

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: debian.eclass
+# @MAINTAINER:
+# Sam James <sam@gentoo.org>
+# @SUPPORTED_EAPIS: 8
+# @BLURB: Convenience eclass for easily using Debian patchsets as upstream
+# @DESCRIPTION:
+# TODO
+
+# @ECLASS_VARIABLE: DEBIAN_PN
+# @PRE_INHERIT
+# @DESCRIPTION:
+# Debian name for the package. Defaults to ${PN}.
+: ${DEBIAN_PN:=${PN}}
+
+# @ECLASS_VARIABLE: DEBIAN_PV
+# @PRE_INHERIT
+# @DESCRIPTION:
+# Debian patch level. Defaults to ${PV/*_p}.
+: ${DEBIAN_PV:=${PV/*_p}}
+
+# @ECLASS_VARIABLE: DEBIAN_ORIG
+# @DEFAULT_UNSET
+# @PRE_INHERIT
+# @DESCRIPTION:
+# Controls whether the "original" upstream sources (unpatched) are fetched
+# from Debian or not. Defaults to no.
```

Comment:
But it defaults to unset, and it seems that `no` will make them fetched actually xP.